### PR TITLE
Account

### PR DESCRIPTION
**Pull request by titor-oopart**
I Implement more features and steps

### CODE REVIEW

File: src/test/java/com/foundation/salesforce/steps/AccountStep.java [hunk 1]
```diff
@@ -109,16 +113,16 @@ public void iFillTheDeleteRequest() {
      */
     @Then("the status code is a number {int}")
     public void theStatusCodeIs(int statusCode) {
```
Comment (alszla) [inline src/test/java/com/foundation/salesforce/steps/AccountStep.java]: Hmm, shouldn't this method be removed?

File: src/test/java/com/foundation/salesforce/steps/AccountStep.java [hunk 2]
```diff
@@ -145,12 +148,112 @@ public void schemaIsValid(String schemaTypeName) {
...
+        }
+        softAssert.assertAll();
+    }
+
+    /**
+     * Verifies response's json body.
+     *
+     * @param bodyFields - Expected fields and values.
+     */
+    @And("the response body includes the following")
```
Comment (alszla) [inline src/test/java/com/foundation/salesforce/steps/AccountStep.java]: Shouldn't this method be removed?

File: src/test/java/com/foundation/salesforce/steps/AccountStep.java [hunk 3]
```diff
@@ -145,12 +148,112 @@ public void schemaIsValid(String schemaTypeName) {
...
+        }
+        softAssert.assertAll();
+    }
+
+    /**
+     * Verifies response headers.
+     *
+     * @param headerFields - Expected headers and values.
+     */
+    @And("the headers include the following")
```
Comment (alszla) [inline src/test/java/com/foundation/salesforce/steps/AccountStep.java]: I think this method is already defined in the common steps.
Shouldn't this method be removed?

File: src/test/java/com/foundation/salesforce/steps/AccountStep.java [hunk 4]
```diff
@@ -145,12 +148,112 @@ public void schemaIsValid(String schemaTypeName) {
...
+        response = restClientApi.delete(ACCOUNT_ENDPOINT + "/" + accountId);
+        response.prettyPrint();
+    }
+
+    /**
+     * This method assert the response message.
+     *
+     * @param bodyFields Expected fields and values.
+     */
+    @And("response contains the following message")
```
Comment (MelissaRoman06) [inline src/test/java/com/foundation/salesforce/steps/AccountStep.java:193]: I think this method is also defined in the common steps.

File: src/test/java/com/foundation/salesforce/steps/AccountStep.java (hunk 1 above)
Comment (titor-oopart) [reply src/test/java/com/foundation/salesforce/steps/AccountStep.java]: done

File: src/test/java/com/foundation/salesforce/steps/AccountStep.java (hunk 2 above)
Comment (titor-oopart) [reply src/test/java/com/foundation/salesforce/steps/AccountStep.java]: done

File: src/test/java/com/foundation/salesforce/steps/AccountStep.java (hunk 3 above)
Comment (titor-oopart) [reply src/test/java/com/foundation/salesforce/steps/AccountStep.java]: done

**Review by MelissaRoman06 — APPROVED** [review]

File: src/test/java/com/foundation/salesforce/steps/CommonSteps.java [hunk 5]
```diff
@@ -63,6 +63,7 @@ public void theStatusCodeIs(int statusCode) {
     @And("the response includes the following")
     public void responseIncludesTheFollowing(Map<String, String> bodyFields) {
         SoftAssert softAssert = new SoftAssert();
+
```
Comment (Cxristian) [inline src/test/java/com/foundation/salesforce/steps/CommonSteps.java]: delete blank space

File: src/test/resources/features/account/CreateAccount.feature [hunk 6]
```diff
@@ -1,30 +1,141 @@
 Feature: Creation funtional and acceptance
```
Comment (Cxristian) [inline src/test/resources/features/account/CreateAccount.feature:2]: tag is missing

File: src/test/java/com/foundation/salesforce/steps/CommonSteps.java (hunk 5 above)
Comment (titor-oopart) [reply src/test/java/com/foundation/salesforce/steps/CommonSteps.java]: done

File: src/test/resources/features/account/CreateAccount.feature (hunk 6 above)
Comment (titor-oopart) [reply src/test/resources/features/account/CreateAccount.feature:2]: done

**Review by alszla — APPROVED** [review]

**Review by Cxristian — APPROVED** [review]

**Pull request merged**